ivy/functional/frontends/torch/tensor.py

```diff
@@ -991,3 +991,7 @@ def fill_(self, value):
 
     def nonzero(self):
         return torch_frontend.nonzero(self)
+
+    @with_unsupported_dtypes({"1.11.0 and below": ("bfloat16",)}, "torch")
+    def multiply(self, other, *, out=None):
+        return torch_frontend.multiply(self._ivy_array, other, out=out)
```

Comment:
I don't think there's a need to do `self._ivy_array` here. Simply calling the frontend function by passing `self` should work as the frontend function's decorators will take care of converting the inputs to and from `ivy arrays` to `frontend array` etc.